Reply/app/src/main/res/layout/fragment_email.xml

```diff
@@ -16,128 +16,173 @@
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto">
 
-    <data>
-        <variable
-            name="email"
-            type="com.materialstudies.reply.data.Email" />
-    </data>
-
+  <data>
+    <variable
+        name="email"
+        type="com.materialstudies.reply.data.Email" />
+  </data>
+
+  <androidx.constraintlayout.widget.ConstraintLayout
+      android:layout_width="match_parent"
+      android:layout_height="match_parent"
+      android:layout_marginStart="@dimen/grid_2"
+      android:layout_marginEnd="@dimen/grid_2"
+      android:layout_marginHorizontal="@dimen/grid_0_5"
+      android:layout_marginVertical="@dimen/grid_4">
+
+    <com.google.android.material.appbar.AppBarLayout
+        android:id="@+id/email_app_bar_layout"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:paddingTop="@dimen/grid_2"
+        android:paddingBottom="@dimen/grid_2"
```

Comment:
Add `android:fitsSystemWindows="true"` as mentioned in above comment

Reply:
To test that all this scrolling stuff works, the email with the image attachments can be opened and scrolled (FYI).

Should look like this I beleive:
![scroll](https://user-images.githubusercontent.com/7459254/137793126-104880b9-2a31-4d7e-a59a-5c52c2b20055.gif)